utils/update_version/update_version.py

```diff
@@ -0,0 +1,204 @@
+
+    if match[2] != new_version_repr:
+        compose_text = re.sub(COMPOSE_VERSION_PATTERN, f'\\g<1>{new_version_repr}\\g<3>', compose_text)
+        with open(compose_file, 'w') as fp:
+            fp.write(compose_text)
+
+        print(f'\u2714 {compose_file} was updated. {match[2]} -> {new_version_repr}\n')
+
+    else:
+        print(f'\u2714 {compose_file} no need to update.')
+
+def update_cvat_version(old_version: str, new_version: Version) -> None:
+    version_file = get_cvat_version_filename()
+    with open(version_file, 'r') as fp:
+        version_text = fp.read()
+
+    new_version_str = f'VERSION = {new_version.cvat_repr()}'
+    version_text = version_text.replace(old_version, new_version_str)
+
+    with open(version_file, 'w') as fp:
+        fp.write(version_text)
+
+    print(f'\u2714 {version_file} was updated. {old_version} -> {new_version_str}\n')
+
+def verify_input(version_types: dict, args: dict) -> None:
+    versions_to_bump = [args[v_type] for v_type in version_types]
+    i = iter(versions_to_bump)
+    if not any(i):
+        raise ValueError(f'One of {list(version_types)} options must be specified')
+    if any(i):
+        raise ValueError(f'Only one of {list(version_types)} options accepted')
+
+def get_cvat_version_filename() -> Path:
+    return Path(__file__).resolve().parents[2] / 'cvat' / '__init__.py'
+
+def get_compose_filename() -> Path:
+    return Path(__file__).resolve().parents[2] / 'docker-compose.yml'
+
+def get_current_version() -> tuple[str, Version]:
+    version_file = get_cvat_version_filename()
+
+    with open(version_file, 'r') as fp:
+        version_text = fp.read()
+
+    match = re.search(CVAT_VERSION_PATTERN, version_text)
+    if not match:
+        raise RuntimeError(f'Failed to find version in {version_file}')
+
+    version = Version(int(match[1]), int(match[2]), int(match[3]), match[4], int(match[5]))
+    return match[0], version
+
+def main() -> None:
+    version_types = {
+        'major': {
+            'action': 'store_true',
+            'help': 'Increment the existing major version by 1',
+        },
+        'minor': {
+            'action': 'store_true',
+            'help': 'Increment the existing minor version by 1',
+        },
+        'patch': {
+            'action': 'store_true',
+            'help': 'Increment the existing patch version by 1',
+        },
+        'prerelease': {
+            'nargs': '?',
+            'const': 'increment',
+            'help': '''Increment prerelease version alpha->beta->rc->final,
+                    Also it's possible to pass value explicitly''',
+        },
+        'prerelease_number': {
+            'action': 'store_true',
+            'help': 'Increment prerelease number by 1',
+        },
+    }
+
+    parser = argparse.ArgumentParser(description='Bump CVAT version')
+
+    for v, v_config in version_types.items():
+        parser.add_argument(f'--{v}', **v_config)
```

Comment:
For better developer experience I think is would be useful to add command that will show current version